tests/unit/sdk/test_utils.py

```diff
@@ -47,22 +49,31 @@ def test_is_valid_uuid() -> None:
     assert is_valid_uuid(uuid.UUID) is False
 
 
+@dataclass
+class ValidURLTestCase:
+    input: Any
+    result: bool
+
+
+VALID_URL_TEST_CASES = [
+    ValidURLTestCase(input=55, result=False),
+    ValidURLTestCase(input="https://", result=False),
+    ValidURLTestCase(input="my-server", result=False),
+    ValidURLTestCase(input="http://my-server", result=True),
+    ValidURLTestCase(input="http://my-server:8080", result=True),
+    ValidURLTestCase(input="http://192.168.1.10", result=True),
+    ValidURLTestCase(input="/test", result=True),
+    ValidURLTestCase(input="/", result=True),
+    ValidURLTestCase(input="http:/192.168.1.10", result=False),
+]
+
+
 @pytest.mark.parametrize(
-    "input,result",
-    [
-        (55, False),
-        ("https://", False),
-        ("my-server", False),
-        ("http://my-server", True),
-        ("http://my-server:8080", True),
-        ("http://192.168.1.10", True),
-        ("/test", True),
-        ("/", True),
-        ("http:/192.168.1.10", False),
-    ],
+    "test_case",
+    [pytest.param(tc, id=str(tc.input)) for tc in VALID_URL_TEST_CASES],
 )
-def test_is_valid_url(input, result) -> None:
-    assert is_valid_url(input) is result
```

Comment:
Refactored this to get rid of the shadowing of the builtin `input()` function, and the need to have `Any` as input in a function signature.